server/coordinator.go

```diff
@@ -211,8 +211,12 @@ func (c *coordinator) addOperator(op Operator) bool {
 	}
 	if op.GetResourceKind() == priorityKind {
 		o, ok := c.operators[regionID]
-		if ok && o.GetResourceKind() == priorityKind {
-			return false
+		if ok {
+			if o.GetResourceKind() == priorityKind {
+				return false
+			}
+			c.limiter.removeOperator(o)
```

Comment:
I guess we should also remove limiter at https://github.com/pingcap/pd/blob/c3ca6537b8945ec4f0040067dd6a16d2af0eefc1/server/coordinator.go#L209